go.mod

```diff
@@ -0,0 +1,12 @@
+module periph.io/x/extra
+
+require (
+	github.com/StackExchange/wmi v0.0.0-20181212234831-e0a55b97c705
+	github.com/go-ole/go-ole v1.2.2 // indirect
+	github.com/google/gousb v0.0.0-20190125150036-d0c05ab7f70d
+	github.com/maruel/ansi256 v1.0.0
+	github.com/mattn/go-colorable v0.0.9
+	github.com/mattn/go-isatty v0.0.4 // indirect
+	golang.org/x/sys v0.0.0-20190129075346-302c3dd5f1cc // indirect
+	periph.io/x/periph v3.4.0+incompatible
```

Comment:
I think it's because periph has a semantic version of v2+ but hasn't got a `go.mod` yet. So I think this will go away with the next release of the core periph. See https://github.com/golang/go/wiki/Modules#can-a-module-consume-a-v2-package-that-has-not-opted-into-modules-what-does-incompatible-mean

Reply:
Thanks for the explanation! Let's update this once v3.5.0 is released.